elide-core/src/main/java/com/yahoo/elide/core/RequestScope.java

```diff
@@ -150,6 +154,7 @@ protected RequestScope(JsonApiDocument jsonApiDocument, RequestScope outerReques
         this.queryParams = Optional.empty();
         this.sparseFields = Collections.emptyMap();
         this.predicates = Collections.emptyMap();
+        this.pagination = Pagination.getDefaultPagination();
```

Comment:
I'm wondering if we want to reject pagination, sort, include, and sparse for all queries except for GET.